server/libs/infra/src/typesense-schemas/face.schema.ts

```diff
@@ -0,0 +1,14 @@
+import { CollectionCreateSchema } from 'typesense/lib/Typesense/Collections';
+
+export const faceSchemaVersion = 1;
+export const faceSchema: CollectionCreateSchema = {
+  name: `faces-v${faceSchemaVersion}`,
+  fields: [
+    // faces
+    { name: 'ownerId', type: 'string', facet: false },
+    { name: 'fileCreatedAt', type: 'string', facet: false, sort: true },
+
+    { name: 'faces', type: 'float[]', facet: false, optional: true, num_dim: 512 },
```

Comment:
This is actually embedding isn't it?

Reply:
Yes, I copied it over from the asset schema. I am unsure how to nest multiple face's embedding in a record for one asset here